app/src/validators/backgroundSchema.ts

```diff
@@ -0,0 +1,23 @@
+import { z } from "zod";
+
+export const BackgroundSchemaValidator = z.object({
+  name: z.string().min(1).max(191),
+  description: z.string().min(1).max(191),
+  isActive: z.boolean().default(false),
+  schema: z.object({
```

Comment:
Extract `schema` to a separate validator, which is also exported and used here, i.e. 

```
export const bgSchema = z.object({
    ocean: z.string().url(),
    ice: z.string().url(),
    dessert: z.string().url(),
    ground: z.string().url(),
    arena: z.string().url(),
    default: z.string().url(),
  })```
  
  Then we can use this validator & its inferred type in the `schema.ts` file